controllers/provider-aws/charts/internal/cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -60,7 +60,7 @@ spec:
         - --cluster-cidr={{ .Values.podNetwork }}
         - --cluster-name={{ .Values.clusterName }}
         - --concurrent-service-syncs=10
-        - --configure-cloud-routes={{ .Values.configureRoutes }}
+        - --configure-cloud-routes=false
```

Comment:
Leftover / why is this changed in the Packet PR?

Reply:
Created https://github.com/gardener/gardener-extensions/pull/80